container.go

```diff
@@ -242,3 +249,12 @@ func (c *ContainerRequest) validateMounts() error {
 	}
 	return nil
 }
+
+// WithFile set the file to be copied before starting a created container
+func (c ContainerRequest) WithFile(f ContainerFile) ContainerRequest {
+	if c.Files == nil {
```

Comment:
Slice is zero-ready. No need to check and init.

Reply:
Yeah, thank you
Forget about that in slices *_*